torch/distributed/rpc/api.py

```diff
@@ -37,8 +37,10 @@
 )
 
 
-logging.basicConfig()
 logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.StreamHandler(sys.stdout))
```

Comment:
This still looks wrong. The user should control all aspects of how logging occurs; in general, libraries should not setup any handlers. Citations: https://docs.python-guide.org/writing/logging/#logging-in-a-library https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library